runtime/src/accounts_db.rs

```diff
@@ -6156,32 +6163,34 @@ impl AccountsDb {
         let len = std::cmp::min(accounts.len(), infos.len());
```

Comment:
Need to adjust or remove the comment right above here.
I imagine I was using the bg thread pool and that was causing a deadlock. We can ignore all of that as you have done by passing a thread pool and making sure we pass the right thread pool.

Reply:
update the comments.
Do we still want that manual batching and `chunk_size`, etc?